src/useGet.test.tsx

```diff
@@ -719,6 +726,48 @@ describe("useGet hook", () => {
       expect(children.mock.calls[3][0].data).toEqual({ id: 2 });
     });
 
+    it("should have a stable refetch function identity", async () => {
+      nock("https://my-awesome-api.fake")
+        .get("/")
+        .reply(200, { id: 1 });
+      const children = jest.fn();
+      children.mockReturnValue(<div />);
+
+      const MyAwesomeComponent = () => {
+        const params = useGet<{ id: number }>({ path: "/", lazy: true });
+        const refetch = params.refetch;
+        React.useEffect(() => {
+          refetch();
+        }, [refetch]);
+        return children(params);
+      };
+
+      // initial fetch
+      render(
+        <RestfulProvider base="https://my-awesome-api.fake">
+          <MyAwesomeComponent />
+        </RestfulProvider>,
+      );
+      await wait(() => expect(children).toBeCalledTimes(3));
+
+      // Initial render before useEffect is called
+      expect(children.mock.calls[0][0].loading).toEqual(false);
+      expect(children.mock.calls[0][0].data).toEqual(null);
+
+      // useEffect initially invoked
+      expect(children.mock.calls[1][0].loading).toEqual(true);
+      expect(children.mock.calls[1][0].data).toEqual(null);
+
+      // finally resolved
+      expect(children.mock.calls[2][0].loading).toEqual(false);
+      expect(children.mock.calls[2][0].data).toEqual({ id: 1 });
+
+      // If the refetch identity weren't stable, this would loop off forever and ever...
+      // you'll know you broke this test if your computer fan goes crazy!
+      expect(children.mock.calls[0][0].refetch).toEqual(children.mock.calls[1][0].refetch);
+      expect(children.mock.calls[0][0].refetch).toEqual(children.mock.calls[2][0].refetch);
```

Comment:
Same as your other PR, I would love to try with `toBe` instead of `toEqual` for this usecase

Reply:
Fixed! https://github.com/contiamo/restful-react/compare/d8998875e04df253c51d9024acb96dd86837cbf6..3f02d9da77cff55d425e1fd05ede3652b4f559b4